evalml/tests/component_tests/test_arima_regressor.py

```diff
@@ -361,4 +375,27 @@ def test_fit_predict_date_index_named_out_of_sample(
         m_clf.fit(X=X, y=y)
         y_pred = m_clf.predict(X=X_test, y=y_test)
 
-    assert (y_pred_sk.to_period("D") == y_pred).all()
+    assert (y_pred_sk == y_pred).all()
+
+
+@pytest.mark.parametrize("freq_num", ["1", "2"])
+@pytest.mark.parametrize("freq_str", ["S", "T", "H", "D", "M", "Y"])
+def test_different_time_units_out_of_sample(freq_str, freq_num):
+    datetime_ = pd.date_range("1/1/1870", periods=20, freq=freq_num + freq_str)
+
+    X = pd.DataFrame(range(20), index=datetime_)
+    y = pd.Series(np.sin(np.linspace(-8 * np.pi, 8 * np.pi, 20)), index=datetime_)
+
+    fh_ = forecasting.ForecastingHorizon(
+        [i + 1 for i in range(len(y[15:]))], is_relative=True
+    )
+
+    a_clf = sktime_arima.AutoARIMA(start_p=2, start_q=2, max_p=2, max_q=2)
```

Comment:
These parameters reduce this test run time from 1m 14s to 20s

Reply:
great call!

Reply:
Do you think it would be the same coverage to just reduce the freq_str to maybe half or so?  I appreciate the thoroughness, but I feel like maybe we can trust sklearn to handle uniformity between the different time periods.